doc/source/getting_started/index.rst

```diff
@@ -648,11 +645,16 @@ the pandas syntax by looking for equivalents from the software you already know:
         </div>
     </div>
 
-Community tutorials
--------------------
+Tutorials
+---------
+
+For a quick introduction to pandas, see :ref:`10 Minutes to pandas<10min>`.
```

Comment:
I personally find "quick overview of pandas functionality" a better description, as I don't it a very "introductory" in the sense of that it doesn't explain much, but mainly shows briefly most areas of pandas functionality.

(but as I said on the issue, I am not fully sure what best to do with or how to frame this 10min page)